tests/boot/test_mcuboot/src/main.c

```diff
@@ -11,10 +11,16 @@
 /* Main entry point */
 int main(void)
 {
+	int err;
+
 	printk("Launching primary slot application on %s\n", CONFIG_BOARD);
 	/* Perform a permanent swap of MCUBoot application */
-	boot_request_upgrade(1);
-	printk("Secondary application ready for swap, rebooting\n");
-	sys_reboot(SYS_REBOOT_COLD);
+	err = boot_request_upgrade(BOOT_UPGRADE_PERMANENT);
+	if (err) {
+		printk("Failed to request upgrade: %d", err);
+	} else {
+		printk("Secondary application ready for swap, rebooting\n");
+		sys_reboot(SYS_REBOOT_COLD);
```

Comment:
Might be worth adding a k_sleep() here for 200ms or so to ensure message is printed (optional)

Reply:
> Might be worth adding a k_sleep() here for 200ms or so to ensure message is printed (optional)

It 's not needed. It is printed on all tested platforms.